### PR TITLE
app manager v2: update page title when updating form name

### PR DESCRIPTION
**Pull request by orangejenny**
https://manage.dimagi.com/default.asp?256330

Debated whether to change `updateDOM` instead of updating `document.title` all over the place, but don't think it's significantly cleaner.

@benrudolph / @biyeun

### CODE REVIEW

File: corehq/apps/app_manager/templates/app_manager/v2/form_view.html [hunk 1]
```diff
@@ -78,13 +78,15 @@
...
+        {% inline_edit_trans form.name langs edit_name_url saveValueName='name' containerClass='h3' iconClass='fcc fcc-app-createform' postSave=postSave %}
       {% elif form.requires_case %}
-        {% inline_edit_trans form.name langs edit_name_url saveValueName='name' containerClass='h3' iconClass='fcc fcc-app-updateform' postSave="function(data) { return hqImport('app_manager/js/app_manager.js').updateDOM(data.update); }" %}
+        {% inline_edit_trans form.name langs edit_name_url saveValueName='name' containerClass='h3' iconClass='fcc fcc-app-updateform' postSave=postSave %}
       {% else %}
-        {% inline_edit_trans form.name langs edit_name_url saveValueName='name' containerClass='h3' iconClass='fa fa-file-o' postSave="function(data) { return hqImport('app_manager/js/app_manager.js').updateDOM(data.update); }" %}
+        {% inline_edit_trans form.name langs edit_name_url saveValueName='name' containerClass='h3' iconClass='fa fa-file-o' postSave=postSvae %}
```
Comment (benrudolph) [inline corehq/apps/app_manager/templates/app_manager/v2/form_view.html]: Svae!

File: corehq/apps/app_manager/static/app_manager/js/form_designer.js [hunk 2]
```diff
@@ -164,6 +164,7 @@ hqDefine("app_manager/js/form_designer.js", function() {
             editDetails.setUpdateCallbackFn(function (name) {
                 $('#formdesigner .fd-content-left .fd-head-text').text(name);
                 $('.variable-form_name').text(name);
+                document.title = name + ' - CommCare HQ';
```
Comment (proteusvacuum) [inline corehq/apps/app_manager/static/app_manager/js/form_designer.js]: So I know this is more of a feature request, but would it be possible to make this a little more informative? 
I'm thinking:
- "{form name} - Form Settings - CommCare HQ"
- "{form name} - Edit Form - CommCareHQ"
- "{module name} - Module Settings - CommcareHQ"

Right now if you have both the settings of the form and the form builder open at the same time it is impossible to know which one is which. With HQ tab soup, this gets a little frustrating.

File: corehq/apps/app_manager/templates/app_manager/v2/form_view.html (hunk 1 above)
Comment (orangejenny) [inline corehq/apps/app_manager/templates/app_manager/v2/form_view.html]: eep, thanks

File: corehq/apps/app_manager/static/app_manager/js/form_designer.js (hunk 2 above)
Comment (orangejenny) [inline corehq/apps/app_manager/static/app_manager/js/form_designer.js]: fair point. @biyeun and I offlined, she offered to take a crack at that in another PR.

Comment (biyeun) [inline corehq/apps/app_manager/static/app_manager/js/form_designer.js]: https://github.com/dimagi/commcare-hq/pull/16746

Comment (benrudolph) [inline corehq/apps/app_manager/static/app_manager/js/form_designer.js]: nice! i think it's these little things that make the experience from good to wow 😮

**Comment by orangejenny** [comment]
closing out in favor of https://github.com/dimagi/commcare-hq/pull/16746

**Pull request closed without merging**